examples/auth.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+# example: python auth.py <SDK-Key>
+
+import json
+import requests
+import sys
+
+# This example demonstrates obtaining an access token and using it to request the
```

Comment:
which option does this correspond to? maybe be explicit in the comment?

Reply:
Will fix it - this demonstrates Issuer & Validator mode.